### DISCUSSION TITLE
v1.2.3-beta6

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#433] [Announcements/公告]
## What's Changed
* 增加始终中文语言选项，新版10.5.53不工作问题修复 by @nov30th in https://github.com/constanline/XQuickEnergy/pull/422
* Master by @constanline in https://github.com/constanline/XQuickEnergy/pull/431
* 需要注意的是，这次变更会根据Download目录下是否存在"useMedia"(文件/目录均可)觉得配置目录使用“/sdcard/Download/xqe”还是“/sdcard/Android/media/com.eg.android.AlipayGphone/xqe”，如果使用了心目里要重新进行文件的配置。但是据我观察，删除掉当前的xqe文件夹使支付宝重新创建xqe可以达到同样的效果(即支付宝并不是不能读取Download目录，而是不能读取旧版支付宝创建的xqe文件夹)，请大家酌情选用。

## New Contributors
* @nov30th made their first contribution in https://github.com/constanline/XQuickEnergy/pull/422
* @constanline made their first contribution in https://github.com/constanline/XQuickEnergy/pull/431

**Full Changelog**: https://github.com/constanline/XQuickEnergy/compare/1.2.3-beta5...v1.2.3-beta6

<hr /><em>This discussion was created from the release <a href='https://github.com/constanline/XQuickEnergy/releases/tag/v1.2.3-beta6'>v1.2.3-beta6</a>.</em>

**Comment by wyfang** [comment]
第三条没看明白啥意思，我能采取什么措施以及会有什么结果？
----
懂了，是“如果使用了新目录里要重新进行文件的配置”。不过实测，我手动删除/sdcard/Download/xqe后，即使Download目录下是存在"useMedia"还是会使用/sdcard/Android/media/com.eg.android.AlipayGphone/xqe。不过无伤大雅。

顺便问下这个xqe目录我看会随着时间容量变大，是里面日志文件变大了吗？以及它是否有最大值？

------
今天突然发现自动恢复到download里的xqe了。

**Reply by JokerPeng6223** [reply]
文件夹内以.log结尾的都是日志，可以定期删除。

**Comment by emmming** [comment]


**Reply by pipi815** [reply]
问题还是没有解决，无法自动收取能量。如何解决

**Comment by pipi815** [comment]
问题仍未解决，无法自动收取能量，miui14.0.2

**Reply by nov30th** [reply]
作者版本里加了一个判断，需手动在Download目录下创建一个useMedia文件夹，程序判断了会使用新目录存放文件。
不行就重启手机再看，啊里佩进程不结束干净不会使用新环境。
目前我手头小米嘎嘎收能量。

之前测试时候删过Download下xqe目录，但总是有奇怪的问题，不是卡白屏就是日志权限之类，我还改了文件后缀，功能正常但日志文件无法访问写入balabala。用了这个目录一劳永逸。

**Reply by pipi815** [reply]
感谢大佬解答

**Comment by rkdawn** [comment]
老铁们，我的大号不会卡启动，分身会卡启动，你们怎么解决的。。

**Reply by huiwin** [reply]
是的我也是这样，小米双开，小号可以正常使用，大号始终白屏，从新把支付宝的版本回退到上一个，并且删除大号的XPE文件夹后可以使用

**Reply by rkdawn** [reply]
> 我也会卡

我的已经解决，可以参考我的解决办法

**Reply by rkdawn** [reply]
> 老铁们，我的大号不会卡启动，分身会卡启动，你们怎么解决的。。

目前已经解决卡白屏，包括但不限于日志空白等问题。
使用环境：小米11U_MIUi14_9.18的开发版，安卓13，支付宝是谷歌play的最新版版本：10.5.50.9600，芝麻粒用的版本：1.2.3-beta6
**方法如下：**
首先正常安装，并且在lsp芝麻粒勾选支付宝后，芝麻粒权限和支付宝权限存储权限先拒绝，再给始终允许；
然后去Download目录下创建useMedia文件夹，这个时候打开支付宝，等待芝麻粒启动，然后点进去森林、庄园等要开启的功能，过一两分钟后去文件夹xqe文件夹是否创建，并且查看目录下下是否缺失文件或者日志文件为0(正常情况下森林日志是0kb空文件，因为芝麻粒那边还没开启)，如果缺失删除xqe文件夹，重复刚才步骤。
如果不缺失，这个时候可以去芝麻粒进行设置相关功能，提示一下，设置的时候，返回到芝麻粒主界面，再进去看设置是否已经成功保存，不会被恢复(因为我发现有些时候，芝麻粒那边写不进去配置，大概率还是存储权限没有给到位的问题)。
设置好了，就把支付宝和芝麻粒后台全部关闭，重新进支付宝，等待一两分钟后，去查看xqe文件夹文件下的各个文件是否正常，特别是日志为0kb的文件是否还存在。
接下来是小号：
大号配置完毕没有异常，就开始用双开自带的双开去分身一个芝麻粒(不要用lsp自带的)，在分身目录下，小米的分身空间名称是/storage/emulated/999/Download/下创建useMedia文件夹，然后按照刚刚的大号步骤一步一步的来，应该是没有问题的，最关键的就是看配置文件是否健全，以及打开芝麻粒能否正常保存配置。

**Comment by xzcgg** [comment]
最新版神奇物种赠送卡片你们能用吗？

**Reply by rkdawn** [reply]
每日任务，送好友卡片？这个没有问题

**Comment by pipi815** [comment]
7点定时启动的功能不能用啊

**Comment by 947814157** [comment]
这个小鸡庄园有了新的游戏这能加进去一下吗大佬，看了还是只有两个游戏，还有支付宝签到里面的一些任务能做吗

**Reply by rkdawn** [reply]
什么新游戏？你是为了饲料？应该每天都用不完啊

**Reply by 947814157** [reply]
当然是为了加速卡呀，现在饲料几乎都用不完了可是加速卡太少了，每天最多五个了不得了，要是饲料几乎快满了你去玩游戏的话几乎都是加速卡我试过了，这样加速卡就很多了

**Reply by rkdawn** [reply]
我还真没留意过，我每天打开支付宝都是随缘，我的加速卡没怎么用过。

**Reply by 947814157** [reply]
加速卡可以在设置里面设置使用呢应该是用了你不知道吧

**Comment by mapleshadow** [comment]
真好！更新了就是好！

**Comment by mapleshadow** [comment]
这个版本卡启动界面，怎么办？怎么解决啊老铁们
